hack/templates/excluder-template

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
Yeah, let's keep everything RPM related in contrib for now.  hack/ if only it's part of the development process.